include_core/omrporterror.h

```diff
@@ -104,7 +104,18 @@
 #define OMRPORT_ERROR_STARTUP_SHMEM (OMRPORT_ERROR_STARTUP_BASE -24)
 #define OMRPORT_ERROR_STARTUP_SHMEM_MOSERVICE (OMRPORT_ERROR_STARTUP_BASE -25)
 #define OMRPORT_ERROR_STARTUP_SYSINFO_MONITOR_INIT (OMRPORT_ERROR_STARTUP_BASE -26)
-#define OMRPORT_ERROR_STARTUP_SIGNAL_TOOLS (OMRPORT_ERROR_STARTUP_BASE -27)
+#define OMRPORT_ERROR_STARTUP_SIGNAL_TOOLS1 (OMRPORT_ERROR_STARTUP_BASE -27)
```

Comment:
This has the potential to break downstream projects (by removing `OMRPORT_ERROR_STARTUP_SIGNAL_TOOLS`).
I suggest we keep it (perhaps as an alias for `OMRPORT_ERROR_STARTUP_SIGNAL_TOOLS1`) which would also eliminate the need to change the Windows implementations of `omrsignal.c`.

Reply:
I only added OMRPORT_ERROR_STARTUP_SIGNAL_TOOLS  two days ago in https://github.com/eclipse/omr/pull/6231

Reply:
Updated.